instrumentation/elasticsearch/elasticsearch-rest-common/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/elasticsearch/rest/ElasticsearchRestNetAttributesExtractor.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.elasticsearch.rest;
+
+import io.opentelemetry.instrumentation.api.instrumenter.net.NetAttributesExtractor;
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.elasticsearch.client.Response;
+
+final class ElasticsearchRestNetAttributesExtractor
+    extends NetAttributesExtractor<String, Response> {
+  @Override
+  public @Nullable String transport(String s) {
+    return null;
```

Comment:
Any reason for not returning `SemanticAttributes.NetTransportValues.IP_TCP`?

Reply:
I initially didn't include it because the original tracer instrumentation didn't set this. You're tight though: the rest client is always using HTTP, so we can safely set that one. Added, thanks!